fab/utils.py

```diff
@@ -159,3 +161,9 @@ def retrieve_cached_deploy_checkpoint():
 def _retrieve_cached(filename):
     with open(filename, 'r') as f:
         return pickle.load(f)
+
+
+def traceback_string():
+    exc_type, exc, tb = sys.exc_info()
+    trace = "".join(traceback.format_tb(tb))
+    return u"Exception: {exc}\n Traceback:\n{trace}".format(exc=exc, t=trace)
```

Comment:
I think this is broken because `{trace}` and `t=trace`. Also, I'd like to suggest a small change:
```python
return u"Traceback:\n{trace}{type}: {exc}".format(
    trace=trace,
    type=exc_type.__name__,
    exc=exc,
)
```